src/ts/Search/Location.tsx

```diff
@@ -27,18 +27,25 @@ export default class Location extends React.Component<any, any> {
 
   componentDidMount() {
     // in case of re-creating component we do not need another script
-    if (document.getElementById('google-maps-script')) {
-      this.initMap();
-      return;
+    if (typeof google !== 'undefined') {
```

Comment:
in this case, I think the variable is declared, so no reference error here. Alas, do not follow my initial comment! Looking at the link you provided, the `typeof` strategy seems to be a good practice.